Install failed - sed error

### Problem description

Jus run the install script

### LunarVim version

1.0.0

### Neovim version (>= 0.6)

NVIM v0.6.1 Build type: Release LuaJIT 2.1.0-beta3 Compiled by brew@HMBRW-A-001-M1-004.local  Features: +acl +iconv +tui See ":help feature-compile"     system vimrc file: "$VIM/sysinit.vim"   fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.6.1/share/nvim"  Run :checkhealth for more info

### Operating system/version

macOS 12.1

### Relevant log output

```shell
LV_BRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh) 
      88\                                                   88\
      88 |                                                  \__|
      88 |88\   88\ 888888$\   888888\   888888\ 88\    88\ 88\ 888888\8888\
      88 |88 |  88 |88  __88\  \____88\ 88  __88\\88\  88  |88 |88  _88  _88\
      88 |88 |  88 |88 |  88 | 888888$ |88 |  \__|\88\88  / 88 |88 / 88 / 88 |
      88 |88 |  88 |88 |  88 |88  __88 |88 |       \88$  /  88 |88 | 88 | 88 |
      88 |\888888  |88 |  88 |\888888$ |88 |        \$  /   88 |88 | 88 | 88 |
      \__| \______/ \__|  \__| \_______|\__|         \_/    \__|\__| \__| \__|

--------------------------------------------------------------------------------
Detecting platform for managing any additional neovim dependencies
--------------------------------------------------------------------------------
Would you like to install LunarVim's NodeJS dependencies?
[y]es or [n]o (default: no) : yes
Installing node modules with npm..
All NodeJS dependencies are successfully installed
--------------------------------------------------------------------------------
Would you like to install LunarVim's Python dependencies?
[y]es or [n]o (default: no) : yes
Verifying that pip is available..
Installing with pip..
Requirement already satisfied: pynvim in ./.local/lib/python3.8/site-packages (0.4.3)
Requirement already satisfied: msgpack>=0.5.0 in ./.local/lib/python3.8/site-packages (from pynvim) (1.0.3)
Requirement already satisfied: greenlet in ./.local/lib/python3.8/site-packages (from pynvim) (1.1.2)
WARNING: You are using pip version 21.1.1; however, version 21.3.1 is available.
You should consider upgrading via the '/Users/user/.pyenv/versions/3.8.10/bin/python3 -m pip install --upgrade pip' command.
All Python dependencies are successfully installed
--------------------------------------------------------------------------------
Would you like to install LunarVim's Rust dependencies?
[y]es or [n]o (default: no) : yes
Installing missing Rust dependency with cargo
    Updating crates.io index
     Ignored package `fd-find v8.3.1` is already installed, use --force to override
warning: be sure to add `/Users/user/.cargo/bin` to your PATH to be able to run the installed binaries
Installing missing Rust dependency with cargo
    Updating crates.io index
     Ignored package `ripgrep v13.0.0` is already installed, use --force to override
warning: be sure to add `/Users/user/.cargo/bin` to your PATH to be able to run the installed binaries
All Rust dependencies are successfully installed
--------------------------------------------------------------------------------
Backing up old /Users/user/.config/lvim to /Users/user/.config/lvim.bak

Number of files: 2 (reg: 1, dir: 1)
Number of created files: 0
Number of deleted files: 0
Number of regular files transferred: 1
Total file size: 0 bytes
Total transferred file size: 0 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.003 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 148
Total bytes received: 38

sent 148 bytes  received 38 bytes  372.00 bytes/sec
total size is 0  speedup is 0.00
--------------------------------------------------------------------------------
Backing up old /Users/user/.local/share/lunarvim to /Users/user/.local/share/lunarvim.bak

Number of files: 143 (reg: 109, dir: 34)
Number of created files: 142 (reg: 109, dir: 33)
Number of deleted files: 0
Number of regular files transferred: 109
Total file size: 1.84M bytes
Total transferred file size: 1.84M bytes
Literal data: 1.84M bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.002 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 1.85M
Total bytes received: 2.24K

sent 1.85M bytes  received 2.24K bytes  3.70M bytes/sec
bash-3.2$ LV_BRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh) 
      88\                                                   88\
      88 |                                                  \__|
      88 |88\   88\ 888888$\   888888\   888888\ 88\    88\ 88\ 888888\8888\
      88 |88 |  88 |88  __88\  \____88\ 88  __88\\88\  88  |88 |88  _88  _88\
      88 |88 |  88 |88 |  88 | 888888$ |88 |  \__|\88\88  / 88 |88 / 88 / 88 |
      88 |88 |  88 |88 |  88 |88  __88 |88 |       \88$  /  88 |88 | 88 | 88 |
      88 |\888888  |88 |  88 |\888888$ |88 |        \$  /   88 |88 | 88 | 88 |
      \__| \______/ \__|  \__| \_______|\__|         \_/    \__|\__| \__| \__|

--------------------------------------------------------------------------------
Detecting platform for managing any additional neovim dependencies
--------------------------------------------------------------------------------
Would you like to install LunarVim's NodeJS dependencies?
[y]es or [n]o (default: no) : yes
Installing node modules with npm..
All NodeJS dependencies are successfully installed
--------------------------------------------------------------------------------
Would you like to install LunarVim's Python dependencies?
[y]es or [n]o (default: no) : yes
Verifying that pip is available..
Installing with pip..
Requirement already satisfied: pynvim in ./.local/lib/python3.8/site-packages (0.4.3)
Requirement already satisfied: msgpack>=0.5.0 in ./.local/lib/python3.8/site-packages (from pynvim) (1.0.3)
Requirement already satisfied: greenlet in ./.local/lib/python3.8/site-packages (from pynvim) (1.1.2)
WARNING: You are using pip version 21.1.1; however, version 21.3.1 is available.
You should consider upgrading via the '/Users/user/.pyenv/versions/3.8.10/bin/python3 -m pip install --upgrade pip' command.
All Python dependencies are successfully installed
--------------------------------------------------------------------------------
Would you like to install LunarVim's Rust dependencies?
[y]es or [n]o (default: no) : yes
Installing missing Rust dependency with cargo
    Updating crates.io index
     Ignored package `fd-find v8.3.1` is already installed, use --force to override
warning: be sure to add `/Users/user/.cargo/bin` to your PATH to be able to run the installed binaries
Installing missing Rust dependency with cargo
    Updating crates.io index
     Ignored package `ripgrep v13.0.0` is already installed, use --force to override
warning: be sure to add `/Users/user/.cargo/bin` to your PATH to be able to run the installed binaries
All Rust dependencies are successfully installed
--------------------------------------------------------------------------------
Backing up old /Users/user/.cache/nvim to /Users/user/.cache/nvim.bak

Number of files: 3 (reg: 2, dir: 1)
Number of created files: 0
Number of deleted files: 0
Number of regular files transferred: 2
Total file size: 0 bytes
Total transferred file size: 0 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.001 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 206
Total bytes received: 57

sent 206 bytes  received 57 bytes  526.00 bytes/sec
total size is 0  speedup is 0.00
--------------------------------------------------------------------------------
Backup operation complete
--------------------------------------------------------------------------------
Cloning LunarVim configuration
Cloning into '/Users/user/.local/share/lunarvim/lvim'...
remote: Enumerating objects: 172, done.
remote: Counting objects: 100% (172/172), done.
remote: Compressing objects: 100% (157/157), done.
remote: Total 172 (delta 4), reused 87 (delta 4), pack-reused 0
Receiving objects: 100% (172/172), 1.65 MiB | 924.00 KiB/s, done.
Resolving deltas: 100% (4/4), done.
--------------------------------------------------------------------------------
Installing LunarVim shim
starting LunarVim bin-installer
bash ./utils/installer/install_bin.sh
sed: -e expression #1, char 47: unknown option to `s'
make: *** [install-bin] Error 1
bash-3.2$
```


### Screenshots

_No response_

--- a/utils/installer/install_bin.sh
+++ b/utils/installer/install_bin.sh
@@ -24,9 +24,9 @@ function setup_shim() {
 
   cp "$src" "$dst"
 
-  sed -e s"@RUNTIME_DIR_VAR@\"${LUNARVIM_RUNTIME_DIR}\"@"g \
-    -e s"@CONFIG_DIR_VAR@\"${LUNARVIM_CONFIG_DIR}\"@"g \
-    -e s"@CACHE_DIR_VAR@\"${LUNARVIM_CACHE_DIR}\"@"g "$src" \
+  sed -e s"#RUNTIME_DIR_VAR#\"${LUNARVIM_RUNTIME_DIR}\"#"g \
+    -e s"#CONFIG_DIR_VAR#\"${LUNARVIM_CONFIG_DIR}\"#"g \
+    -e s"#CACHE_DIR_VAR#\"${LUNARVIM_CACHE_DIR}\"#"g "$src" \
     | tee "$dst" >/dev/null
 
   chmod u+x "$dst"

